g4f/Provider/__init__.py

```diff
@@ -55,5 +56,6 @@
     "Yqcloud",
     "Equing",
     "FastGpt",
+    "Wuguokai"
```

Comment:
comma missed

Reply:
@johnd0e @xtekky ahh sorry my bad

Reply:
@johnd0e @xtekky 
i make another PR to my mistake : https://github.com/xtekky/gpt4free/pull/845